### ISSUE TITLE
[Feature Request] - Support for mouse cursors in sokol_imgui

### ISSUE THREAD

**Issue by raegnar** [floooh/sokol#459]
It would be nice to have mouse cursors supported when using sokol with imgui.(ImGuiBackendFlags_HasMouseCursors)

**Comment by raegnar** [comment]
So I tried applying some of the mousecursor code I found in `imgui_impl_osx.mm` and it sort of worked. However it seems to flick and appear unstable. Not really sure what is causing this.

Flickering cursor from sokol + imgui
![flickeringCursor](https://user-images.githubusercontent.com/593220/105643976-046f6e80-5e48-11eb-8442-db3eea632e06.gif)

Stable cursor from GLFW + imgui (both interacting with the same UI element with the same settings)
![stableCursor](https://user-images.githubusercontent.com/593220/105643980-06d1c880-5e48-11eb-8bf6-5e4f217b8b12.gif)
